sdk/android/analytics.md

```diff
@@ -6,6 +6,13 @@ title: Analytics
 {% endif %}
 {% assign media_base = base_dir | append: page.dir | append: "media" %}
 
+<br/>
+
+**Note**
+This guide shows how to build an app using AWS Mobile SDK for Android and the Amplify CLI toolchain.
+To use our new, preview developer experience with new Amplify Libraries for Android, [click here.](../../android/analytics)
```

Comment:
maybe another place where we can explain the difference to our customers? why do I want to use Amplify over the SDK